packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -86,17 +87,18 @@ function transformMetricKind(kind: OTMetricKind): MetricKind {
     case OTMetricKind.UP_DOWN_COUNTER:
     case OTMetricKind.VALUE_OBSERVER:
     case OTMetricKind.UP_DOWN_SUM_OBSERVER:
+    case OTMetricKind.VALUE_RECORDER:
       return MetricKind.GAUGE;
     default:
-      // TODO: Add support for OTMetricKind.ValueRecorder
-      // OTMetricKind.Measure was renamed to ValueRecorder in #1117
       return MetricKind.UNSPECIFIED;
   }
 }
 
 /** Transforms a OpenTelemetry ValueType to a StackDriver ValueType. */
-function transformValueType(valueType: OTValueType): ValueType {
-  if (valueType === OTValueType.DOUBLE) {
+function transformValueType(valueType: OTValueType, point: OTPoint): ValueType {
+  if (isDistributionValue(point.value) || isHistogramValue(point.value)) {
```

Comment:
I'm not familiar with the JS SDK, but looking through this code, `isDistribution()` will be true for `ValueRecorder`. Is that right?

I'm confused because `transformValue()` will then throw on L242.

Reply:
Yeah, that's correct. I've opened a separate PR to address the support for `Distribution` and `Histogram` type metrics to be exported here: https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/125

When testing with those changes, I've gotten the `ValueRecorder` to correctly export.